admin/database/database.go

```diff
@@ -255,18 +258,19 @@ type Project struct {
 	Description          string
 	Public               bool
 	Region               string
-	GithubURL            *string   `db:"github_url"`
-	GithubInstallationID *int64    `db:"github_installation_id"`
-	Subpath              string    `db:"subpath"`
-	ProdBranch           string    `db:"prod_branch"`
-	ProdVariables        Variables `db:"prod_variables"`
-	ProdOLAPDriver       string    `db:"prod_olap_driver"`
-	ProdOLAPDSN          string    `db:"prod_olap_dsn"`
-	ProdSlots            int       `db:"prod_slots"`
-	ProdTTLSeconds       *int64    `db:"prod_ttl_seconds"`
-	ProdDeploymentID     *string   `db:"prod_deployment_id"`
-	CreatedOn            time.Time `db:"created_on"`
-	UpdatedOn            time.Time `db:"updated_on"`
+	GithubURL            *string        `db:"github_url"`
+	GithubInstallationID *int64         `db:"github_installation_id"`
+	Subpath              string         `db:"subpath"`
+	ProdBranch           string         `db:"prod_branch"`
+	ProdVariables        Variables      `db:"prod_variables"`
+	ProdOLAPDriver       string         `db:"prod_olap_driver"`
+	ProdOLAPDSN          string         `db:"prod_olap_dsn"`
+	ProdSlots            int            `db:"prod_slots"`
+	ProdTTLSeconds       *int64         `db:"prod_ttl_seconds"`
+	ProdDeploymentID     *string        `db:"prod_deployment_id"`
+	Tags                 pq.StringArray `db:"tags"`
```

Comment:
We don't use `lib/pq`, we use `jackc/pgx` for Postgres. Does this work? If yes, you can just put `[]string` here. If not, should use `pgtype.Array[string]`.

Reply:
I tried both ways you suggested, only works if we use pq.StringArray or custom impl of Tags